packages/babel/package.json

```diff
@@ -32,7 +36,7 @@
     "test:ts": "tsc types/index.d.ts test/types.ts --noEmit"
   },
   "files": [
-    "dist",
+    "dist/**/*.{js,json}",
```

Comment:
I think that I would prefer using just `dist` here - it's IMHO easier to manage as that directory is a built artifact anyway.

If that's the convention used by the whole monorepo then let be it. But there is a potential that this would have to be tweaked after the source code migrates to TS

Reply:
I configured the build process to emit sourcemaps by default for debugging, but I do not want to emit them. Admittedly, this could also be done via a negative pattern.